src/renderingManager.js

```diff
@@ -71,7 +70,7 @@ export function newRenderingManager(win, environment) {
     let windowLocation = window.location;
     let parsedUrl = utils.parseUrl(pubUrl);
     let publisherDomain = parsedUrl.protocol + '://' + parsedUrl.host;
-    let adServerDomain = pubAdServerDomain || GOOGLE_IFRAME_HOSTNAME;
+    let adServerDomain = pubAdServerDomain || win.location.hostname;
```

Comment:
Any particular reason why inside some function, sometime uses `window`, sometime uses `win`?

Reply:
no reason.  updated to make it consistent.